hack/run-ci-e2e-test.sh

```diff
@@ -14,7 +14,13 @@ WMCO_PATH_OPTION=""
 export CGO_ENABLED=0
 
 get_WMCO_logs() {
-  oc logs -l name=windows-machine-config-operator -n $WMCO_DEPLOY_NAMESPACE --tail=-1 >> "$ARTIFACT_DIR"/wmco.log
+  retries=0
+  until [ $retries -gt 4 ] || oc logs -l name=windows-machine-config-operator -n $WMCO_DEPLOY_NAMESPACE --tail=-1 >> "$ARTIFACT_DIR"/wmco.log
+  do
+	let retries+=1
+	echo "failed to get WMCO logs"
+	sleep 5
```

Comment:
Is it possible to indent?

Reply:
I just replaced the tabs with spaces, didnt realize they were there.
 This file is split between 4 and 2 spaces, but the majority of it is 2 space indenting, so chose 2 for this function.